MPSGameLift/Scripts/export_gamelift_server_package.py

```diff
@@ -172,46 +179,54 @@
                          "--assetListFile", engine_asset_list_path]) != 0):
         quit()
 
-# Add all the required exe's, dll's, and asset pak files into a folder to upload to GameLift
-gamelift_package_cache_dir = os.path.join(gamelift_package_folder_name, "Cache", "pc")
-gamelift_package_gems_dir = os.path.join(gamelift_package_folder_name, "Gems", "AWSCore")
+def create_exe_package(new_package_folder_name, exe_name):
+    # Add all the required exe's, dll's, and asset pak files into a folder to upload to GameLift
+    package_cache_dir = os.path.join(new_package_folder_name, "Cache", "pc")
 
-# Delete the old server package
-if os.path.exists(gamelift_package_folder_name):
-    shutil.rmtree(gamelift_package_folder_name)
+    # Delete the old server package
+    if os.path.exists(new_package_folder_name):
+        shutil.rmtree(new_package_folder_name)
 
-# Create the folders
-os.makedirs(gamelift_package_cache_dir, exist_ok=True)
-os.makedirs(gamelift_package_gems_dir, exist_ok=True)
+    # Create the folders
+    os.makedirs(package_cache_dir, exist_ok=True)
+
+    # Copy .exe and .dll files to GameLiftWindowsServerPackage directory
+    build_dir = os.path.join(monolithic_build_folder, "bin", "release")
+    for file_name in os.listdir(build_dir):
+        file_path = os.path.join(build_dir, file_name)
+        if os.path.isfile(file_path) and file_name.lower().endswith((exe_name.lower(), '.dll')):
+            shutil.copy(file_path, new_package_folder_name)
```

Comment:
maybe use `shutil.copy2` to preserve the metadata of the exe and dlls

Reply:
Fixed!